pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -64,19 +77,37 @@ class VmOverviewTabLibvirt extends React.Component {
     }
 
     close() {
-        this.setState({ showModal: false });
+        this.setState({ showVcpuModal: false, showBootOrderModal: false });
+    }
+
+    openVcpu() {
+        this.setState({ showVcpuModal: true });
     }
 
-    open() {
-        this.setState({ showModal: true });
+    openBootOrder() {
+        this.setState({ showBootOrderModal: true });
     }
 
     render() {
-        const { vm, dispatch, config } = this.props;
+        const runningVmChanged = () => {
+            const activeDevices = getBootOrderDevices(vm);
+            const inactiveDevices = getBootOrderDevices(vm.inactiveXML);
+
+            if (inactiveDevices.length !== activeDevices.length) {
+                // number of bootable devices has changed
+                return true;
+            } else {
+                // only order of devices has changes
```

Comment:
It might be also possible that nothing has changed no?

Reply:
In matters of bootable devices.

Reply:
Yeah right. But if nothing changed, then !every() will return false, which is correct.
Comment in my code is misleading, I will change that.